plugins/europeana/search.js

```diff
@@ -36,22 +36,22 @@ export const unquotableFacets = [
 ];
 
 // Thematic collections available via the `theme` parameter.
-// Order is significant as it will be reflected on search results.
-export const thematicCollections = [
-  'ww1',
-  'archaeology',
-  'art',
-  'fashion',
-  'industrial',
-  'manuscript',
-  'map',
-  'migration',
-  'music',
-  'nature',
-  'newspaper',
-  'photography',
-  'sport'
-];
+// Order is significant as it will be reflected in the search results filter.
+export const thematicCollections = new Map([
```

Comment:
Uses a Map instead of a basic Object to ensure order of insertion is respected on iteration.